Makefile

```diff
@@ -193,3 +193,16 @@ download-logs:
 tensorboard:
 	poetry install --only tensorboard
 	marian-tensorboard --offline -f logs/*.log
+
+# Run the GitHub pages Jekyll theme locally.
+serve-docs:
+	echo "This command requires"
+	echo "  rbenv: https://github.com/rbenv/rbenv"
+	echo "  rbenv install 3.2.2"
```

Comment:
I think I had some issues with ruby installation on mac os, not sure. This is better than nothing but would it be easier to take some off the shelve docker image and just do docker run here? Maybe something like this https://hub.docker.com/r/bretfisher/jekyll

Reply:
ok